builder/virtualbox/ovf/config.go

```diff
@@ -87,13 +89,12 @@ func NewConfig(raws ...interface{}) (*Config, []string, error) {
 	errs = packer.MultiErrorAppend(errs, c.VBoxManagePostConfig.Prepare(&c.ctx)...)
 	errs = packer.MultiErrorAppend(errs, c.VBoxVersionConfig.Prepare(&c.ctx)...)
 
-	if c.SourcePath == "" {
-		errs = packer.MultiErrorAppend(errs, fmt.Errorf("source_path is required"))
```

Comment:
I think it has something to do with `DownloadableURL` returning an error on empty strings, so I think it errors there instead, but I'll verify.